sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -73,14 +75,18 @@ public ConfigurationClient(Uri endpoint, TokenCredential credential)
         /// <param name="endpoint">The <see cref="Uri"/> referencing the app configuration storage.</param>
         /// <param name="credential">The token credential used to sign requests.</param>
         /// <param name="options">Options that allow configuration of requests sent to the configuration store.</param>
+        /// <remarks> The <paramref name="credential"/>'s Azure Active Directory audience is configurable via the <see cref="ConfigurationClientOptions.Audience"/> property.
+        /// If no token audience is set, the client attempts to set an audience based on the provided <paramref name="endpoint"/>,
```

Comment:
I may tune this a bit to just mention that we default to Public if not specified.   We really want them to set the audience if non-public.   We'll use the super-safe-service-team-guaranteed fallbacks as a courtesy, but I'd rather not document that.